internal/brokers/auth/consts.go

```diff
@@ -18,8 +18,8 @@ const (
 var Replies = []string{Granted, Denied, Cancelled, Retry, Next}
 
 const (
-	// SessionModeAuth is the name of the authentication session.
-	SessionModeAuth = "auth"
-	// SessionModePasswd is the name of the passwd session.
-	SessionModePasswd = "passwd"
+	// SessionModeAuthenticate is used when the session is for user authentication.
+	SessionModeAuthenticate = "auth"
```

Comment:
Maybe for the same spirit, this should be `SessionModeLogin`? Since it does both authentication and session initialization...

Reply:
I love it! Done!